kanidmd/src/lib/tracing_tree/event_tag.rs

```diff
@@ -0,0 +1,104 @@
+use super::subscriber::EventTagSet;
+use std::convert::TryFrom;
+
+#[derive(Debug, Clone, Copy)]
+pub enum KanidmEventTag {
```

Comment:
Is there reason you're reimplementing LogTag in `src/lib/audit.rs` which seems to have very similar values?

Reply:
i think he plans to drop the content of audit.rs, which I'm okay with.

Reply:
I did it so I could have similar tags on my test branch, but either this or the `LogTag` type is going to go. What's important is we have some `enum` type that implements `tracing_tree::subscriber::EventTagSet`